.github/workflows/ci.yml

```diff
@@ -92,5 +90,31 @@ jobs:
           actual: ${{ needs.test-build-ok-with-custom-image-name.outputs.image_tag }}
       - uses: nick-fields/assert-action@v2
         with:
-          expected: "custom_image_name.tar"
+          expected: "custom_image_name"
           actual: ${{ needs.test-build-ok-with-custom-image-name.outputs.image_artifact }}
+
+  test-push-ok:
+    needs: test-build-ok-with-artifact
+    uses: ./.github/workflows/push.yml
+    with:
+      environment: dev
+      dockerfile: "fixture/Dockerfile.artifact"
+
```

Comment:
Should we test for the presence of an image in GCR?

Reply:
Absolutely. Work in progress here: https://github.com/entur/gha-docker/pull/25/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR114